CoreNotesView/src/au/gov/asd/tac/constellation/views/notes/NotesViewPane.java

```diff
@@ -722,8 +788,12 @@ private void createNote(final NotesViewEntry newNote) {
 
             // If the note to be created is in edit mode, ensure it is created
             // with the correct java fx elements
-            noteInformation = new VBox(DEFAULT_SPACING, newNote.getEditMode() ? titleText : titleLabel,
-                    newNote.getEditMode() ? contentTextArea : contentLabel);
+            if (newNote.getEditMode()) {
+                noteInformation = new VBox(DEFAULT_SPACING, titleText, contentTextArea);
+            } else {
+                noteInformation = new VBox(DEFAULT_SPACING, containerPane);
+            }
```

Comment:
can you make this a ternary statement? can be combined with the declaration that way